src/components/buoy-telemetry/DateSlider.vue

```diff
@@ -0,0 +1,99 @@
+<template>
```

Comment:
I forget if we needed the date slider later and we're just ignoring it for now. Or if this isn't needed at all for buoy-telemetry and can be pulled out.

Reply:
We do need the DateSlider, but are working around it for now to get one timepoint of data displayed.